resources/views/auth/login.blade.php

```diff
@@ -17,21 +17,15 @@
         </div>
 
         <div class="flex items-center justify-between">
-            <x-forms.inputs.checkbox name="remember" id="remember">
-                Remember me
-            </x-forms.inputs.checkbox>
+            <x-forms.inputs.checkbox name="remember" id="remember">Remember me</x-forms.inputs.checkbox>
 
             <div class="text-sm">
-                <a href="{{ route('password.forgot') }}" class="font-medium text-lio-600 hover:text-lio-500">
-                    Forgot your password?
-                </a>
+                <a href="{{ route('password.forgot') }}" class="font-medium text-lio-600 hover:text-lio-500"> Forgot your password? </a>
```

Comment:
Spaces around the words here. Happens in quite a few places. Interestingly enough it doesn't happens all the time.